src/csharp/BUILD-INTEGRATION.md

```diff
@@ -105,6 +105,39 @@ button](https://stackoverflow.com/a/9770061). Click on it, and choose "Add as
 link". If you do not select this option, Visual Studio will copy files to the
 project directory instead.
 
+#### My .proto files have same filename in different folders
+
+Starting from Grpc.Tools version 2.31, protocol buffers compilation preserves original folder structure for generated files. Eg.
```

Comment:
That's not actually what https://github.com/grpc/grpc/pull/22869 does. We do use directory hashes to disambiguate the generated files.

`ProjectFolder/obj/hash_generated_from_relative_path/Greet.cs`

@Falco20019  to avoid confusion, maybe we should close this PR and you can open a new one with up-to-date description of what https://github.com/grpc/grpc/pull/22869 actually does?

Reply:
@jtattermusch I have just described the behaviour I see while using Grpc.Tools version `2.31.0-dev202007191122`. I agree that it would be better that @Falco20019 will add relevant documentation as he is an author not me.

Reply:
@jtattermusch I can try to have a look in the future. Right now, I'm sadly stuck in a couple of topics and then out of office. So I can't create a PR in the next 4 weeks. I will keep this PR bookmarked for now.

Reply:
@jtattermusch Sorry, this seems to got lost on my schedule. I will mark it again as TODO.